src/unpoly/viewport.sass

```diff
@@ -14,6 +14,8 @@ up-bounds
 body.up-scrollbar-away
   &, html:has(>&)
     overflow-y: hidden !important
+    @supports (overflow-y: clip)
```

Comment:
Would leaving out the `@supports` do the same?

```css
overflow-y: hidden !important
overflow-y: clip !important
```

Reply:
You're right, it absolutely would. 👍